docs/component_branding.md

```diff
@@ -29,9 +29,16 @@ To add colours to a component, modify the component to follow the example below.
 
 Note that the helper must be called for each element that needs a border or link colour applying. This allows for flexibility if one is required but not the other.
 
+The `brand_border_color` method can also be applied to the parent element of a component, in the situation where the component consists of only one element. This is currently not possible with `brand_color` as there is no requirement for this.
```

Comment:
What do you mean here by "parent element"?

Reply:
The highest level element of the component - i.e. .gem-c-component-name. This was a bit tricky because I'd already come up with the branding model where I'd assumed parent element with children, but of course the heading component is only one element, so I had to redo the branding a little to be able to apply the branding classes to the parent element as well, not just the parent and child.

Does this need to be written more clearly? Was 'parent element' in this context confusing?

Reply:
Ah ok, when I read "parent element" I think of an element immediately before another element. So in this case I thought you meant something that might wrap the `gem-c-` element..

Reply:
I've rewritten this a little to hopefully make it clearer - I can see why it was confusing.